.github/workflows/jdbc-tests-db-collation.yml

```diff
@@ -0,0 +1,186 @@
+name: JDBC Tests DB COLLATION
+on: [push, pull_request]
+
+jobs:
+  run-babelfish-jdbc-tests:
+    env:
+      INSTALL_DIR: psql
+      DATABASE_COLLATION_NAME: bbf_unicode_cp1_ci_ai
+    runs-on: ubuntu-20.04
+    steps:
+      - uses: actions/checkout@v2
+        id: checkout
+
+      - name: Install Dependencies
+        id: install-dependencies
+        if: always()
+        uses: ./.github/composite-actions/install-dependencies
+
+      - name: Build Modified Postgres
+        id: build-modified-postgres
+        if: always() && steps.install-dependencies.outcome == 'success'
+        uses: ./.github/composite-actions/build-modified-postgres
+        with:
+          install_dir: 'psql'
+          code_coverage: 'yes'
```

Comment:
- install_dir: The default value of `install_dir` as `psql` in composite action `build-modified-postgres` is `psql`, so no need of specifying the same from here.
- code_coverage: we are not using code coverage summary of this workflow, hence generating a code coverage summary for this workflow is redundant.

Reply:
Updated